imagemounter/volume_system.py

```diff
@@ -279,6 +305,11 @@ def _detect_mmls_volumes(self, vstype='detect'):
                 except Exception as e:
                     logger.exception("Failed executing mmls command")
                     raise SubsystemError(e)
+            # since mmls cannot determine the partition type for exFAT superfloppy formatting; check for this.
+            elif self._detect_exfat_superfloppy():
```

Comment:
It has been removed from the _detect_mmls_volumes method, but I have not removed it from the _detect_parted_volumes method.  As mentioned in the comment re: _make_exfat_superfloppy I have one example where the exFAT superfloppy is not properly detected.  Now, with the other changes it only happens when parted is specified as the detection method.  When a wiped drive (thumb drive in this case) is formatted using Linux (Ubuntu 16.04/Mint 18.1) not in a volume but as the superfloppy, parted detects a msdos partition table, but does not detect the filesystem at offset 0.  It detects the offset at 32 and returns the description as 'free'.

The presence of the 'EXFAT' in the header of the image file will differentiate between a drive that does not have any volumes and one that has exFAT.  The following condition is used:

`if self._detect_exfat_superfloppy():`
`    volume.offset = 0`
`    volume.flag = 'alloc'`
`    volume.info['fsdescription'] = 'exfat'`
`else:`
`    volume.flag = 'unalloc'`

The thumb drive was wiped and formatted with the following command:
`# mkfs.exfat -n '8GB_exFAT' /dev/sdX`

Perhaps there is better way to format the drive, but both Linux and Win 10 pick it up and read/write to it fine.  The image file is small so I can make it available if you would like to take a look at it.

As far as moving the method call somewhere else, I don't know where it could go and still be used to differentiate between a truly "'description == 'free'" condition and one that is only erroneously detected.